instrumentation/opentelemetry-instrumentation-boto/src/opentelemetry/instrumentation/boto/__init__.py

```diff
@@ -204,3 +204,46 @@ def _patched_auth_request(self, original_func, instance, args, kwargs):
             args,
             kwargs,
         )
+
+
+def add_span_arg_tags(span, endpoint_name, args, args_names, args_traced):
+    def truncate_arg_value(value, max_len=1024):
+        """Truncate values which are bytes and greater than `max_len`.
```

Comment:
Sorry this is my bad, I should've mentioned that I just copied and pasted this code into this file because the `botocore` instrumentation no longer needs it.

`boto` and `botocore` both used to share it, but now I removed it out of `botocore` and left it to only be used by `boto`.

[Removing the import in boto](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/150/files#diff-5696731c7113e6a7f09773bb9233a2227cf642a1cb6a0c5aa45c4fb330da735bL54)

[Removing the function from botocore because it doesn't need it](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/150/files#diff-05ef2bbeb79bb2d595bf8a3f7fa2d475d0f34f80804e5e1ba0d9a253d2c0e0e3L176)

I can see us refactoring the `boto` instrumentation in a follow up PR?

I could have made the diff simpler by leaving it in `botocore`, but `botocore` doesn't need it. Do you have an opinion on this?

Reply:
It makes sense to address your comments regardless I imagine, I can make it part of another PR if we want.

Created an issue here: https://github.com/open-telemetry/opentelemetry-python-contrib/issues/186